server/server.go

```diff
@@ -140,13 +140,7 @@ func (s *Server) loadContainer(id string) error {
 		return err
 	}
 
-	var img *pb.ImageSpec
-	image, ok := m.Annotations[annotations.Image]
-	if ok {
-		img = &pb.ImageSpec{
-			Image: image,
-		}
-	}
+	img, _ := m.Annotations[annotations.Image]
```

Comment:
I had assumed that img would be set to "" if an image annotation wasn't found since it's the default string value, but I can change that